docs/deployment.md

```diff
@@ -0,0 +1,93 @@
+# Deployment
+
+## Deploying locally
+
+Your function can run locally either by starting it through Visual
+Studio or by running `dotnet run` from the command line. If your
+project contains multiple functions, you must specify which one you
+want to host either via command line arguments or environment
+variables. The port to serve on can also be specified via the
+command line or environment variables.
+
+See the [README](../README.md) for configuration details.
+
+## Deploying to Cloud Run
+
+Running the Functions Framework is equivalent to running a regular
+ASP.NET Core application as far as infrastructure is concerned. This
+means you can run functions on Cloud Run with ease.
+
+There are detailed instructions for C# available as part of the
+Cloud Run documentation:
+
+- [Quickstart for C#](https://cloud.google.com/run/docs/quickstarts/build-and-deploy#c)
+- [Codelab for C#](https://codelabs.developers.google.com/codelabs/cloud-run-hello-csharp/index.html)
+
+The instructions apply to functions in exactly the same way as
+normal, with the same Dockerfile configuration. The `PORT`
+environment variable will be supplied by Cloud Run. If your project
+contains multiple functions, the simplest approach is to specify an
+environment variable in your Dockerfile, with a line like this:
+
+```text
+ENV FUNCTION_TARGET HelloFunctions.Function
+```
+
+## Deploying to Google Cloud Functions
+
+> **NOTE:**  
+> The `dotnet3` runtime is currently available on an "allow
```

Comment:
If there are instructions on how to get whitelisted, or at least a note on who gets whitelisted, we can avoid people asking.

Reply:
+1. This is a bit ominous.

Reply:
Added a bit here. PTAL.